scripts/runhttpd

```diff
@@ -76,4 +82,5 @@ configure_restart_on_certificate_update "${IRONIC_TLS_SETUP}" httpd "${IRONIC_CE
 # Set up inotify to kill the container (restart) whenever cert of httpd for /shared/html/<redifsh;ilo> path change
 configure_restart_on_certificate_update "${IRONIC_VMEDIA_TLS_SETUP}" httpd "${IRONIC_VMEDIA_CERT_FILE}"
 
-exec /usr/sbin/httpd -DFOREGROUND
+exec /usr/sbin/httpd -DFOREGROUND \
```

Comment:
nit: I don't think the line break is justified here  :)

Reply:
Agree